Makefile

```diff
@@ -15,6 +15,8 @@ build: fmtcheck go-lint
 generate: tools
 	# generate object tests
 	tools/tools object-generator --mode tests --in ./pkg/... --out xxgenerated_object_test.go
+	# generate GetIdentifier methods for objects
+	tools/tools object-generator --mode object-generator --in ./pkg/... --out xxgenerated_object_identifier.go
```

Comment:
object-generator in mode object-generator sounds strange ... why not the suggested "runtime"?